ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml

```diff
@@ -99,6 +99,18 @@ tests:
     test:
     - chain: cucushift-sdn-migration-ovn
     workflow: cucushift-installer-rehearse-aws-ipi
+- as: e2e-aws-ipi-ovn-hypershift-p2
```

Comment:
We are migrating tests from Jenkins into Prow, and the related profile are retired then in Jenkins.
Currently we are doing this for 4.12 (migrate one profile to Prow, then retire the profile in Jenkins).
We haven't started the process for 4.11 yet (We plan to do that after we are done for 4.12), we should not add it for 4.11 at the moment so that we have less duplicate tests exist in both Jenkins and Prow (there are 4.11 tests exist in Prow because they are added before we start 4.12).